realm/realm-library/src/androidTest/java/io/realm/OrderedRealmCollectionTests.java

```diff
@@ -221,6 +221,14 @@ public void first_emptyCollection() {
     }
 
     @Test
+    public void first_withDefault() {
+        AllJavaTypes defaultObject = collection.get(0);
```

Comment:
Nitpick, but instead of getting an object from a collection defined outside this method you could just create a standalone object.